modules/installation-initializing.adoc

```diff
@@ -471,14 +471,17 @@ additionalTrustBundle: |
 The value must be the contents of the certificate file that you used for your mirror registry. The certificate file can be an existing, trusted certificate authority, or the self-signed certificate that you generated for the mirror registry.
 
 ifdef::aws+restricted[]
-.. Define the subnets for the VPC to install the cluster in:
+.. Define the subnets for the VPC to install the cluster in, as in the following example:
 +
 [source,yaml]
 ----
-subnets:
-- subnet-1
-- subnet-2
-- subnet-3
+platform:
+  aws:
+    vpc:
+      subnets:
+        - id: subnet-<id1>
+        - id: subnet-<id2>
+        - id: subnet-<id3>
```

Comment:
Idk if this would look better or worse, but maybe an underscore between ID and the numbers?
```suggestion
        - id: subnet-<id_1>
        - id: subnet-<id_2>
        - id: subnet-<id_3>
```
Up to you